model/waterfall.go

```diff
@@ -92,7 +92,7 @@ func getBuildDisplayNames(match bson.M) bson.M {
 }
 
 // This pipeline matches on versions that have a build with an ID or display name that matches variants
-func getBuildVariantFilterPipeline(ctx context.Context, variants []string, match bson.M, projectId string) ([]bson.M, error) {
+func getBuildVariantFilterPipeline(ctx context.Context, variants []string, match bson.M, projectId string, bvSearchCutoff int) ([]bson.M, error) {
```

Comment:
Why is this named bvSearchCutoff? Does it have to do with buildVariants? (The usage below of finding a version by revision order number with it makes me confused which is why I ask). Is it worth modifying the func comment to explain the parameter?

Reply:
Good points, I renamed the param and added a comment!